### PR TITLE
Simplify ERT to get index of CU from KDS

### PR DESCRIPTION
**Pull request by stsoe**
KDS now writes selected CU into the cmd packet cumask while
writing command to ERT command queue.  This simplifies processing
in ERT firmware

### CODE REVIEW

**Comment by stsoe** [comment]
retest this please

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/mb_scheduler.c [hunk 1]
```diff
@@ -1479,8 +1464,17 @@ ert_start_cmd(struct xocl_ert *xert, struct xocl_cmd *xcmd)
 	SCHED_DEBUG_PACKET(ecmd, cmd_packet_size(xcmd));
 
 	// write packet minus header
-	SCHED_DEBUGF("++ slot_idx=%d, slot_addr=0x%x\n", xcmd->slot_idx, slot_addr);
-	xocl_memcpy_toio(xert->cq_base + slot_addr + 4, ecmd->data, (cmd_packet_size(xcmd) - 1) * sizeof(u32));
+	if (cmd_type(xcmd) == ERT_CU && !XOCL_DSA_IS_VERSAL(xcmd->xdev)) {
```
Comment (mamin506) [inline src/runtime_src/core/pcie/driver/linux/xocl/subdev/mb_scheduler.c]: Nots: Soft kernel command is using type ERT_DEFAULT.
This change is safe for soft kernel.

**Comment by stsoe** [comment]
retest this please

**Comment by maxzhen** [comment]
retest this please

**Comment by stsoe** [comment]
retest this please

**Pull request merged**